Core/Interfaces/Security/Annotation/AnnotationsLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ArtoxLab\Bundle\ClarcRbacBundle\Core\Interfaces\Security\Annotation;
+
+use Doctrine\Common\Annotations\Reader;
+use ReflectionClass;
+use ReflectionException;
+use RuntimeException;
+
+final class AnnotationsLoader implements AnnotationsLoaderInterface
+{
+    public function __construct(private Reader $reader)
```

Comment:
PHP 7.3 does not support the constructor property promotion
https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion